data/sqlite3/sqlite3_test.go

```diff
@@ -0,0 +1,59 @@
+package sqlite3
+
+import (
+	"database/sql"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/google/uuid"
+
+	"github.com/haydenwoodhead/burner.kiwi/data"
+)
+
+func TestSQLite3(t *testing.T) {
+	dburl := os.Getenv("DATABASE_URL")
```

Comment:
I don't think there's any need to get the db url here. We can just pass it a file name directly. Plus it breaks stuff in travis where Postgres also uses this env var value.

Reply:
Could also clean it up after finishing testing.